specs/vscode_extensions.table

```diff
@@ -0,0 +1,17 @@
+table_name("vscode_extensions")
+description("Lists all vscode extensions.")
+schema([
+    Column("id", TEXT, "Extension id"),
+    Column("version", TEXT, "Extension version"),
+    Column("path", TEXT, "Extension path"),
+    Column("publisher", TEXT, "Publisher Name"),
+    Column("installed_at", BIGINT, "Installed Timestamp"),
+    Column("prerelease", INTEGER, "Pre release version"),
```

Comment:
I'm not convinced of the utility of this column -- does it add anything additionally useful beyond `version`? In my VSCode extensions I couldn't find any that returned true for this.

Reply:
it works if you have any prerelease extensions installed. on my machine, I have a prerelease version of the Python extension installed and it does show up.
![image](https://github.com/osquery/osquery/assets/32922990/1957eb27-389b-4b58-8404-a88a8ad38cd8)

Reply:
I think we should at least make sure that the data is returned consistently. On my machine there are some extensions that return empty rather than 0:

```
osquery> select * from vscode_extensions limit 3;
W1102 14:17:05.602226 1147323968 vscode_extensions.cpp:55] Could not read vscode extensions.json from '/Users/zwass/.vscode-server/extensions/extensions.json'
          id = cschlosser.doxdocgen
     version = 1.4.0
        path = /Users/zwass/.vscode/extensions/cschlosser.doxdocgen-1.4.0
   publisher = Christoph Schlosser
installed_at = 1645810251454
  prerelease = 0
         uid = 501

          id = editorconfig.editorconfig
     version = 0.16.4
        path = /Users/zwass/.vscode/extensions/editorconfig.editorconfig-0.16.4
   publisher = EditorConfig
installed_at = 1635895302700
  prerelease =
         uid = 501

          id = jeff-hykin.better-cpp-syntax
     version = 1.17.2
        path = /Users/zwass/.vscode/extensions/jeff-hykin.better-cpp-syntax-1.17.2
   publisher = Jeff Hykin
installed_at = 1670820541875
  prerelease = 0
         uid = 501
         ```